docs/environment.rst

```diff
@@ -134,14 +134,28 @@ Google Cloud Storage
 .. envvar:: TENSORSTORE_GCS_HTTP_URL
 
    Specifies to connect to an alternative server in place of
-   ``https://storage.googleapis.com``.  Note that the normal Google oauth2
+   ``https://storage.googleapis.com`` for the ``gcs`` driver (HTTP-based).  
+   Note that the normal Google oauth2 credentials *are* included in requests, 
+   and therefore only trusted servers should be used.
+
+.. envvar:: TENSORSTORE_GCS_GRPC_ENDPOINT
+
+   Specifies to connect to an alternative gRPC endpoint in place of the default
+   ``dns:///storage.googleapis.com`` or ``google-c2p:///storage.googleapis.com`` 
+   (on GCE VMs) for the ``gcs_grpc`` driver. Note that the normal Google oauth2
    credentials *are* included in requests, and therefore only trusted servers
    should be used.
 
 .. envvar:: TENSORSTORE_GCS_REQUEST_CONCURRENCY
 
    Specifies the concurrency level used by the shared Context
    :json:schema:`Context.gcs_request_concurrency` resource. Defaults to 32.
+   
+.. envvar:: TENSORSTORE_GCS_GRPC_CHANNELS
```

Comment:
This is just a documentation improvement: this environment variable was used but not listed in the docs.